tests/management/test_refresh_job_postings_data.py

```diff
@@ -0,0 +1,80 @@
+# -*- coding: utf-8 -*-
```

Comment:
Do these test cover the entirety of `taxonomy/management/commands/refresh_job_postings_data.py` ?

Reply:
I did add one more test case, let me know if have anything else in mind?